.changeset/clean-owls-sparkle.md

```diff
@@ -0,0 +1,6 @@
+---
+'@leafygreen-ui/radio-box-group': patch
+'@leafygreen-ui/select': patch
```

Comment:
Technically, changing the name of the exported type from `Props` to `SelectProps` would be a breaking change

Reply:
It shouldn't be, since `Props` wasn't being exported from the package.